packages/browser/src/tracing/browserTracingIntegration.ts

```diff
@@ -132,6 +132,13 @@ export interface BrowserTracingOptions {
    */
   traceXHR: boolean;
 
+  /**
+   * Flag to disable tracking of long-lived streams, like server-sent events (SSE) via fetch.
+   *
+   * Default: false
+   */
+  trackFetchStreamPerformance: boolean;
```

Comment:
We should mention here that people should not use this in case they have live-streams or other very very long streams.